src/main/java/com/google/api/codegen/config/RetryCodesConfig.java

```diff
@@ -92,6 +92,41 @@ public static RetryCodesConfig create(
     return retryCodesConfig;
   }
 
+  public static RetryCodesConfig create(GrpcGapicRetryMapping retryMapping, String service) {
+    RetryCodesConfig retryCodesConfig = new RetryCodesConfig();
+    Map<String, ImmutableList<String>> codesDefMap = new HashMap<>();
+    Map<String, String> methodNamesMap = new HashMap<>();
+    ImmutableMap<String, String> methodCodesMap = retryMapping.methodCodesMap();
+    ImmutableMap<String, RetryCodesDefinitionProto> codesMap = retryMapping.codesDefMap();
+
+    methodCodesMap
+        .keySet()
+        .stream()
+        .filter(name -> name.startsWith(service))
+        .forEach(
```

Comment:
nit: Generally if there are multiple lines in the `forEach` block, for loops are preferred. The code works for sure.

Reply:
I will change it! Thanks for the tip!

Reply:
Done.